.github/workflows/triage_handle_new_comments.yml

```diff
@@ -0,0 +1,33 @@
+name: 'Handle Issue/PR Comment Workflow'
+on:
+  # makes this workflow reusable
+  workflow_call:
+    secrets:
+      ADD_TO_TRIAGE_BOARD_TOKEN:
+        required: true
+
+  issue_comment:
+    types: [created]
+
+jobs:
+  handle-comment-scenarios:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          repository: 'cypress-io/release-automations'
+          ref: 'master'
+          ssh-key: ${{ secrets.WORKFLOW_DEPLOY_KEY }} 
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - name: Run comment_workflow.js Script
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.ADD_TO_TRIAGE_BOARD_TOKEN }}
+          script: |
+            const script = require('./scripts/triage/comment_workflow.js')
+            script.handleComment(github, context);
```

Comment:
```suggestion
            await script.handleComment(github, context);
```

Reply:
why does this need to be awaited?

Reply:
Otherwise the process might close before the comment is correctly handled.

> You can also use async functions in this manner, as long as you await it in the inline script.

https://github.com/actions/github-script#run-a-separate-file-with-an-async-function